py/desispec/workflow/calibration_selection.py

```diff
@@ -61,7 +61,7 @@ def determine_calibrations_to_proc(etable, do_cte_flats=True,
     ## return nothing so that we swiftly exit and wait for more data.
     if np.sum(exptypes == 'dark') >= 1 and np.sum(exptypes == 'arc') >= 5 \
             and np.sum(exptypes == 'flat') >= 12 \
-            and (np.sum(exptypes == 'cteflat') >= 3 or not do_cte_flats
+            and ((np.sum(is_cte_1s) >= 1) or not do_cte_flats
```

Comment:
Done.  When we added the additional 3s and 10s flats, we purposefully put those before the 1s flat so that it could remain the "trigger" for launching the pipeline.  If we added/removed other flats in the future, we should do the same (put them between the 120s fiberflats and the 1s CTE flat).